### PR TITLE
Adding tests on a live server (via docker).

### PR DESCRIPTION
**Pull request by josejulio**
TODO:
- [ ] Make alerts specs aware of VCR_UPDATE

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,16 @@
+sudo: required
+
 language: ruby
 rvm:
   - 2.0
```
Comment (pilhuhn) [inline .travis.yml:5]: I wonder (and this is nothing for this PR specifically) if we should drop explicit support for 2.0 (and perhaps even 2.1)

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
   - 2.1
   - 2.2
   - 2.3.0
+env:
+  - RUN_ON_LIVE_SERVER=1 VCR_UPDATE=1 SKIP_SSL_WITHOUT_CERTIFICATE_TEST=1 \
```
Comment (pilhuhn) [inline .travis.yml]: Only cosmetic (and no requirement to change): I would find one per line more easy to quickly see what is set.

Comment (josejulio) [reply .travis.yml]: You are right on that, I'll update it.

File: .travis.yml (hunk 1 above)
Comment (josejulio) [reply .travis.yml:5]: Not sure, if those versions are no longer supported/used on MIQ it would be good to trim them out.

Comment (pilhuhn) [reply .travis.yml:5]: Well, that client is not only for MiQ. MiQ required 2.2.2+

**Comment by pilhuhn** [comment]
Thank you @josejulio

**Pull request merged**